cgmes-gl-conversion/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
General remark: shouldn't we update all header license to 2020 ?

Reply:
that file was coded in 2019 so we should keep 2019.